<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MinionWidget</class>
 <widget class="QWidget" name="MinionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MinionWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="characteristicLayout">
     <item>
      <widget class="QGroupBox" name="statsBox">
       <property name="title">
        <string>Stats</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QCheckBox" name="statsShuffleBetweenMinions">
          <property name="toolTip">
           <string/>
          </property>
          <property name="text">
           <string>Shuffle Between Minions</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="statsShuffleStats">
          <property name="toolTip">
           <string/>
          </property>
          <property name="text">
           <string>Shuffle Stats</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="specialMagicBox">
       <property name="title">
        <string>Special / Magic</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <property name="formAlignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <item row="0" column="0">
         <widget class="QCheckBox" name="specialMagicRandom">
          <property name="toolTip">
           <string/>
          </property>
          <property name="text">
           <string>Random</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="specialMagicPool">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="text">
           <string>Pool</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="spawnBox">
     <property name="font">
      <font>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>Spawn</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0">
      <item>
       <widget class="QGroupBox" name="spawnStoryBox">
        <property name="title">
         <string>Story</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QCheckBox" name="spawnStoryRandomRealtime">
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>Random (Realtime)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="spawnStoryRandomPremade">
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>Random (Premade)</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="spawnEternalCorridorBox">
        <property name="title">
         <string>Eternal Corridor</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QCheckBox" name="spawnEternalCorridorRandom">
           <property name="font">
            <font>
             <bold>false</bold>
            </font>
           </property>
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>Random</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="spawnDreamBox">
        <property name="title">
         <string>Dream</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <widget class="QCheckBox" name="spawnDreamRandom">
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>Random</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QFormLayout" name="spawnDreamLayout">
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QCheckBox" name="spawnDreamCustom">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
             <property name="toolTip">
              <string/>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="iconSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="spawnDreamCombo">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>110</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="toolTip">
              <string/>
             </property>
             <property name="maxVisibleItems">
              <number>10</number>
             </property>
             <property name="sizeAdjustPolicy">
              <enum>QComboBox::AdjustToContents</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="spawnKorisBox">
        <property name="title">
         <string>Koris</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QCheckBox" name="spawnKorisRandom">
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>Random</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QFormLayout" name="spawnKorisLayout">
           <item row="0" column="0">
            <widget class="QCheckBox" name="spawnKorisCustom">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
             <property name="toolTip">
              <string/>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="spawnKorisCombo">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>110</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="toolTip">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QGroupBox" name="appearanceBox">
     <property name="title">
      <string>Appearance</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,1,1">
      <item>
       <widget class="QCheckBox" name="appearanceRandomNewMinion">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Random New Minion</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="appearanceModelBox">
        <property name="title">
         <string>Model</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QCheckBox" name="appearanceModelRandom">
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>Random</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="appearanceTextureBox">
        <property name="title">
         <string>Texture</string>
        </property>
        <layout class="QFormLayout" name="formLayout_2">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <property name="labelAlignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="formAlignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <item row="0" column="0">
          <widget class="QCheckBox" name="appearanceTextureRandom">
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>Random</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="appearanceTextureIncludeCompatible">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>+Include Compatible</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="appearanceTextureRandomColor">
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>Random Color</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="appearanceGrowthSizeBox">
        <property name="title">
         <string>Growth Size</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QCheckBox" name="appearanceGrowthSizeShuffle">
           <property name="text">
            <string>Shuffle</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="appearanceGrowthSizeInvert">
           <property name="text">
            <string>Invert</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
